src/slack_api_util.ts

```diff
@@ -17,6 +18,13 @@ slackAPI.defaults.headers.post[
   'Authorization'
 ] = `Bearer ${process.env.SLACK_BOT_ACCESS_TOKEN}`;
 
+axiosRetry(slackAPI, {
+  // This function is passed a retryCount which can be used to
+  // define custom retry delays.
+  retryDelay: () => 1000 /* millisecs between retries*/,
```

Comment:
Your PR description says 2000ms but this is 1000ms -- which one did you intend?

Reply:
Good catch, thanks! Changed to 2000